tests/resources/test_first_quantization.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for functions needed for estimating the number of logical qubits and non-Clifford gates
+for quantum algorithms in first quantization using a plane-wave basis.
+"""
+import pytest
+import pennylane as qml
+from pennylane import numpy as np
+
+
+@pytest.mark.parametrize(
```

Comment:
Tests look good, but again, let's raise some warnings if the arguments don't have the right form (positive numbers, integers) and test the warnings are raised properly